tasks/repo-Suse.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Add the Docker Zypper repository.
+  community.general.zypper_repository:
+    repo: "{{ docker_repo['zypper_repo'] }}"
+    name: "{{ docker_repo['name'] }}"
+    state: present
+    auto_import_keys: yes
+    overwrite_multiple: yes
+    runrefresh: yes
+  when: ansible_distribution == "openSUSE Leap"
```

Comment:
This is only needed when version channel is Leap because Tumbleweed channel already has the latest version. The version available in Leap has proven to be quite buggy (ironically because Leap is supposed to be the stable branch)